packages/react/src/components/RadioGroup/index.tsx

```diff
@@ -157,22 +165,23 @@ RadioGroup.propTypes = {
       id: PropTypes.string.isRequired,
       label: PropTypes.string.isRequired,
       labelDescription: PropTypes.string
-    })
```

Comment:
Name does appear to be optional from the typescript types.

Reply:
I think GitHub might have collapsed the relevant line. This change isn't for `name` but for the object in the `radios` array

Reply:
Yup, my bad 🤦